crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -609,8 +616,14 @@ en:
           available if your setup requires them.
         ceph_repo: 'Ceph Repositories'
         ha_repo: 'HA Repositories'
+        ceph_repo_present: 'Ceph Repositories are available.'
+        ceph_repo_missing: 'Ceph Repositories are missing, please provide them.'
+        ha_repo_present: 'HA Repositories are available.'
+        ha_repo_missing: 'HA Repositories are missing, please provide them.'
+        no_additional_repos: 'No additional Repositories needed.'
```

Comment:
Hrm, I just realized, but I think we want "repositories" (lowercase) for everything that is a sentence not in camel case.